external-crates/move/crates/move-analyzer/trace-adapter/src/trace_utils.ts

```diff
@@ -55,11 +56,12 @@ interface JSONTraceType {
     ref_type?: JSONTraceRefType
 }
 
-type JSONTraceRuntimeValueType = boolean | number | string | JSONTraceRuntimeValueType[] | JSONTraceCompound;
+type JSONTraceRuntimeValueType = {
```

Comment:
The `JSONTraceRuntimeValueType` was a bit unfortunate. Perhaps we should use this opportunity to rename it into something like `JSONTraceMoveValue` or `JSONTraceSerializableMoveValue`